### ISSUE TITLE
Possible to download torrent option?

### ISSUE THREAD

**Issue by eightydee** [sbussetti/deluge-siphon#23]
I have multiple seedboxes. I would like the option to perhaps add multiple selection and the option to download locally or to a folder on the seedbox.

**Comment by sbussetti** [comment]
Yeah, totally do-able.  I can add the ability to have multiple servers saved in your options, and then select which server to send to when you add.  Will do.

**Comment by juicy11** [comment]
Is this option available, have not noticed? Running version 0.70.6.

**Comment by sbussetti** [comment]
@juicy11 @eightydee this will be in the next version I'm working on now. (0.70.7)

**Comment by sbussetti** [comment]
@juicy11 soon?  check the commit history, I've been getting a lot done lately.

**Comment by ipat8** [comment]
Sorry to necro this, but this would be a great feature to have and I haven't seen it show up yet. Any chance for and update?
